plugins/destination/snowflake/client/client_test.go

```diff
@@ -24,5 +25,7 @@ func TestPlugin(t *testing.T) {
 			SkipMigrateOverwrite:      true,
 			SkipMigrateOverwriteForce: true,
 			SkipMigrateAppendForce:    true,
-		})
+		},
+		schema.WithTestSourceSkipIntervals(),
```

Comment:
can't it be defaulted to string/json?

Reply:
I think not at least not easily. The go driver for snowflake is not great. anyway no one is using it at the moment.